R/ppc-distributions.R

```diff
@@ -47,9 +48,16 @@
 #'    quantiles. `y` is overlaid on the plot either as a violin, points, or
 #'    both, depending on the `y_draw` argument.
 #'   }
+#'   \item{`ppc_pit_ecdf()`}{
+#'    The ECDF of the empirical PIT values of `y` computed with respect to the
+#'    corresponding `yrep` values. `100 * prob`% central simultaneous confidence
+#'    intervals are provided to asses if ´y´ and ´yrep´ originate from the same
+#'    distribution. The PIT values can also be provided directly as `pit`.
+#'    See Säilynoja et al. for more details.}
```

Comment:
I suggest also to add a year to disambiguate if you later have another paper to be used in bayesplot as well.